### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Olekk17**
[DEMO LINK](https://olekk17.github.io/react_goods-selector/)

### CODE REVIEW

**Review by Anastasia2403 — APPROVED** [review]
GJ 👍

File: src/App.tsx [hunk 1]
```diff
@@ -15,71 +16,86 @@ export const goods = [
...
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  handleAddButtonClick = (good: string) => (
+    this.setState({ selectedGood: good })
```
Comment (BondarenkoDmitriy) [inline src/App.tsx:31]: it can be done with one function

**Review by BondarenkoDmitriy — APPROVED** [review]
gj

File: src/App.tsx [hunk 2]
```diff
@@ -15,71 +16,86 @@ export const goods = [
...
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  handleClearButtonClick = () => this.setState({ selectedGood: '' });
```
Comment (SerhiiKirik) [inline src/App.tsx:28]:
```suggestion
  handleClear = () => this.setState({ selectedGood: '' });
```

File: src/App.tsx [hunk 3]
```diff
@@ -15,71 +16,86 @@ export const goods = [
...
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={this.handleClearButtonClick}
```
Comment (SerhiiKirik) [inline src/App.tsx:49]: let's make a destructuring from  this

File: src/App.tsx [hunk 4]
```diff
@@ -15,71 +16,86 @@ export const goods = [
...
-          </td>
+        <table className="table">
+          <tbody>
+            {goods.map(good => (
+              <tr
+                key={good}
+                data-cy="Good"
+                className={classNames(
+                  selectedGood === good && 'has-background-success-light',
+                )}
```
Comment (SerhiiKirik) [inline src/App.tsx:63]: Check this [doc](https://www.npmjs.com/package/classnames)
Also, create variable & reuse it
```suggestion
                className={classNames({
                  'has-background-success-light': isGoodSelected;
                }
```

**Review by SerhiiKirik — CHANGES_REQUESTED** [review]
Great 🔥 
Let's improve your solution 🧑‍💻

**Review by SerhiiKirik — APPROVED** [review]
LGTM 🔥

**Pull request closed without merging**